### PR TITLE
Modify player design

### PR DESCRIPTION
**Pull request by fenglish**
### Appy new layout
Screenshot 2019-07-12 at 16 56 14

### Optimised the css grid for expanded player

BEFORE --- 3 x 6
Screenshot 2019-07-12 at 15 00 29

AFTER --- 1 x 7
Screenshot 2019-07-12 at 14 57 42

### CODE REVIEW

File: components/x-audio/src/components/Players.jsx [hunk 1]
```diff
@@ -11,8 +11,12 @@ import { TimeRemaining } from './TimeRemaining';
...
 import { Image } from './Image';
 
-const Title = ({ text }) => <Marquee className={classNameMap('audio-player__info__title')} text={text} />
-const SeriesName = ({ text }) => <Marquee className={classNameMap('audio-player__info__series-name')} text={text} />
+const Title = ({ text }) => (
+	<div className={classNameMap('audio-player__title__marquee')}>
+		<Marquee className={classNameMap('audio-player__info__title')} text={text} />
+	</div>
+);
+const SeriesName = ({ text }) => <div className={classNameMap('audio-player__info__series-name')}><span title={text}>{text}</span></div>;
```
Comment (benbarnett) [inline components/x-audio/src/components/Players.jsx]: Does the series name not need to be a marquee anymore?

File: components/x-audio/src/components/Players.jsx [hunk 2]
```diff
@@ -11,8 +11,12 @@ import { TimeRemaining } from './TimeRemaining';
 import { Marquee } from './Marquee';
 import { Image } from './Image';
 
-const Title = ({ text }) => <Marquee className={classNameMap('audio-player__info__title')} text={text} />
-const SeriesName = ({ text }) => <Marquee className={classNameMap('audio-player__info__series-name')} text={text} />
+const Title = ({ text }) => (
+	<div className={classNameMap('audio-player__title__marquee')}>
```
Comment (benbarnett) [inline components/x-audio/src/components/Players.jsx]: Totally up to you, but I would say the original class name (`audio-player__title`) was sufficient here.

Comment (fenglish) [reply components/x-audio/src/components/Players.jsx]: 👍

File: components/x-audio/src/components/Players.jsx (hunk 1 above)
Comment (fenglish) [reply components/x-audio/src/components/Players.jsx]: 📝 The reasons why we remove the marquee from `SeriesName`
- When  both `Title` and `SeriesName`'s scrolling marquee animation happen, it is impossible to read either of them.
- Initially it was not asked by Designer.
- `SeriesName` is tend to be short. It's an edge case that `SeriesName` needs the animation.

We will put back it when requested.

**Comment by fenglish** [comment]
Thank you @benbarnett !

**Pull request merged**